paymentsheet/src/test/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporterTest.kt

```diff
@@ -664,8 +666,10 @@ class DefaultEventReporterTest {
         PaymentConfiguration.clearInstance()
         // Would crash if it tries to read from the uninitialized PaymentConfiguration
         DefaultEventReporter(
+            mock(),
```

Comment:
Should we use robolectric or similar instead (`ApplicationProvider.getApplicationContext()`)?

Reply:
https://github.com/stripe/stripe-android/pull/10550/commits/ba0e5459a91655a44d6b4eb5e5469007814613f7